### ISSUE TITLE
NTR  "left/right pattern formation" & "determination of left/right asymmetry"

### ISSUE THREAD

**Issue by gocentral** [geneontology/go-ontology#10899]
Hi,

I am wondering why these two terms, "left/right pattern formation" & "determination of left/right symmetry", don't have a relationship to each other. There is also a term for "left/right axis specification" which is a part_of "left/right pattern formation. 

I am working on annotating ciliary genes and am noticing that between the three terms, MGI has annotations for ciliary genes and Hedgehog signaling to all three of them. While some of this may be due to differences in the experiments shown, I suspect some of this is due to confusion between these terms.

GO:0007368 - "determination of left/right symmetry" 
Def: The establishment of an organism's body plan or part of an organism with respect to the left and right halves. The pattern can either be symmetric, such that the halves are mirror images, or asymmetric where the pattern deviates from this symmetry.
- 79 annotations for 59 genes in MGI

GO:0060972 - "left/right pattern formation"
Def: The pattern specification process that results in the subdivision of the left/right axis in space to define an area or volume in which specific patterns of cell differentiation will take place or in which cells interpret a specific environment.
- 10 annotations for 10 genes in MGI

GO:0070986 - "left/right axis specification"
Def: The establishment, maintenance and elaboration of the left/right axis. The left/right axis is defined by a line that runs orthogonal to both the anterior/posterior and dorsal/ventral axes. Each side is defined from the viewpoint of the organism rather of the observer (as per anatomical axes).
- 8 annotations for 8 genes

Then under "determination of left/right symmetry", there are a bunch of terms for things like "determination of heart left/right asymmetry" (GO:0061371) and several other "determination of ____ left/right asymmetry" terms. In the other branch, under "left/right pattern formation", there is a term for "embryonic heart tube left/right pattern formation", but no other anatomy specific terms.

I am a little confused by the lack of connection between "left/right pattern formation" & "determination of left/right symmetry".

Also, there are a bunch of "determination of ___ left/right asymmetry" terms under "determination of left/right symmetry", but there is no term for "determination of "left/right asymmetry". Should there be?

thanks,

-Karen

Reported by: krchristie

Original Ticket: [geneontology/ontology-requests/10713](https://sourceforge.net/p/geneontology/ontology-requests/10713)

**Comment by RLovering** [comment]
I have 6 annotations to lung development waiting for the lung asymmetry term requested in 2014 to enable these 6 annotations to be revised to a more specific term.
Thanks
Ruth

**Comment by cmungall** [comment]
It can help to relabel tickets. This ticket is phrased like a question, not a NTR. I also added the label 'new term request'

**Comment by ukemi** [comment]
@krchristie and I were just discussing the fate of these terms the other day. It seems to us that a single left/right patterning process determined by nodal flow specifies handedness in the embryo. The handedness defects seen in organs are downstream consequences of the original process going awry. Do you know of specific cases where a separate process is used for each organ? For example, do you know of mutants in which only a given organ is affected and the general L/R asymmetry is otherwise normal?

**Comment by krchristie** [comment]
@ukemi Based on what I have seen in annotating, my impression is that there is a single process where generation of nodal flow is required for breaking left/right symmetry in order to form asymmetric organs. 

Once nodal flow and generation of asymmetry is disrupted, there is often some variability in the phenotypes i.e. the offspring have a mix of situs inversus (complete reversal), situs solitus (normal patterning, and heterotaxy (randomization). However, generally multiple organs are affected.

After working through that Noctua model of the paper we discussed previously, I lean towards the position of NOT having the organ specific left/right patterning terms, and only having the top level term as I am not aware of there being any thing specific to the individual organ systems.

**Comment by pgaudet** [comment]
@krchristie Do you want to take this ticket ?

**Comment by krchristie** [comment]
Yes, that would make sense @pgaudet

**Comment by raymond91125** [comment]
Similar to
"determination of radial symmetry" partof "radial pattern formation"
Perhaps it should be that 
"determination of left/right symmetry" partof "left/right pattern formation"

**Comment by ukemi** [comment]
Are left/right patterns always symmetric? I think radial ones are.

**Comment by ukemi** [comment]
I guess if they are symmetric, then the determination of the symmetry is always a part of the pattern formation.

**Comment by raymond91125** [comment]
I thought even if the result is asymmetry, determination is still a part of the normal pattern formation.
"determination of left/right symmetry" Definition: The establishment of an organism's body plan or part of an organism with respect to the left and right halves. The pattern can either be symmetric, such that the halves are mirror images, or asymmetric where the pattern deviates from this symmetry.

**Comment by ukemi** [comment]
Yup. I think you are correct.

**Comment by krchristie** [comment]
> I thought even if the result is asymmetry, determination is still a part of the normal pattern formation.
> "determination of left/right symmetry" Definition: The establishment of an organism's body plan or part of an organism with respect to the left and right halves. The pattern can either be symmetric, such that the halves are mirror images, or asymmetric where the pattern deviates from this symmetry.

My question is really more practical. How would an annotator tell the difference between pattern formation and determination of left/right symmetry? For example, when Paola and I were working on cilia, we decided that in practice there was no difference between "cilium assembly" and "cilium morphogenesis". While both phrases were used in the literature, they referred to the same thing, so we merged them into one term with the label "cilium assembly".

I think we need to think about whether there is something different going on that distinguishes these terms, or not. 

Along those lines, as @ukemi and I have discussed previously, there are lots of terms for things like "**determination of x left/right asymmetry**", where x = "heart", "digestive tract", "nervous system", etc, as if there were different processes determining asymmetry in different systems, but in all the organisms I'm familiar with the initiation of asymmetry (mice, humans, zebrafish), there is only one symmetry breaking process which is required for correct asymmetric development of all of these organs.

Basically, I think there are many unnecessary and unhelpful terms in this branch.

**Comment by raymond91125** [comment]
I would have interpreted the term
"left/right axis specification": not about (a)symmetry, but about addressing (proximal - distal) on the left or right axis.
"determination of left/right symmetry": determining between symmetry vs asymmetry, for some mater regulator genes, perhaps those like nodal or the cilia that controls nodal distribution.
"left/right pattern formation": executes the patterning of left or right parts. axis specification and determination of symmetry are parts.

If we can agree on some distinguishing definitions, we may then begin reviewing existing annotations. 

"determination of x left/right asymmetry" may just be of incomplete observations even if all x are of the same mechanisms. But one may not know that a priori (beyond vertebrates, e.g.).

**Comment by pgaudet** [comment]
That makes sense to me.

**Comment by raymond91125** [comment]
@krchristie Would the added relationship and the chain of discussions sufficient to solve this issue? I am not claiming that the beginnings and ends of these processes are clear.

**Comment by raymond91125** [comment]
> I have 6 annotations to lung development waiting for the lung asymmetry term requested in 2014 to enable these 6 annotations to be revised to a more specific term. Thanks Ruth

@RLovering I cannot find a ticket for the term request. Could you find it or submit a new one? Thanks.

**Comment by krchristie** [comment]
In the long term, I think it might be best to NOT have the "determination of x left/right asymmetry" for various x where x = some specific organ and instead deal with the asymmetric development of different organs via GO-CAMs. In the numerous different organs that develop asymmetrically in the mouse, it is very clear that there is only 1 symmetry breaking event which then effects multiple downstream specific developmental processes in many different organs. 

I realize that my knowledge is based largely on vertebrate embryonic development, but because symmetry breaking is a very early event in embryonic development that utilizes the very ancient structure of the cilium, I would suspect that this is a very ancient mechanism that provides an early signal in patterning, whether that is manifested as radial symmetry or breaking of the left/right symmetrical axis in development.

However, I certainly do not have the funding to put any thought towards reorganizing these terms, nor do I think that GO-CAM models are parsed out in ways that really serve current needs for basic annotation. Thus, in the interest of practicality for immediate annotation, we should probably just create the term for lung that @RLovering is requesting.

**Comment by raymond91125** [comment]
@krchristie point well taken. Is it OK to close this ticket for now? Or we can leave it open for possible further discussions, of course. It's up to you. Thanks.

**Comment by RLovering** [comment]
Hi 
Please will you create the lung term as there do seem to be lots of terms for the other organs eg heart, liver, intestine, nervous system etc
Thanks
Ruth

**Comment by pgaudet** [comment]
https://github.com/geneontology/go-ontology/issues/10899#issuecomment-124912547

**Comment by pgaudet** [comment]
+[Term]
+id: GO:0140969
+name: determination of lung left/right asymmetry
+namespace: biological_process
+def: "Determination of the asymmetric location of the lungs with respect to the left and right halves of the organism." [GOC:BHFL, GOC:rl, PMID:19944405, PMID:20007846, PMID:21131974]
+is_a: GO:0007368 ! determination of left/right symmetry
+relationship: part_of GO:0007507 ! heart development
+property_value: term_tracker_item https://github.com/geneontology/go-ontology/issues/10899 xsd:anyURI
+created_by: pg
+creation_date: 2023-01-03T17:36:31Z
+


@RLovering
